### PR TITLE
Use example markdown generated from code instead of hand-written

### PR DESCRIPTION
**Pull request by trevtrich**

### CODE REVIEW

**Comment by travi** [comment]
just thinking out loud, the first thing that comes to mind here is how much configuration of specific plugins should be explained in the `project-scaffolder` specifically vs linking to the plugin examples, like https://github.com/travi/javascript-scaffolder#example.

beyond the chances of the plugin details getting out of date as things evolve, i also wonder what the right balance is of suggesting which plugins to install/configure if not the right fit for whoever might consume this tool. `semantic-release`, for example, is still working [to take steps back](https://github.com/semantic-release/semantic-release/issues/1260) from the strongly encouraged defaults.

there is obviously benefit to show the basics of including a language plugin, but i dont know what the right balance of detail is. i wonder if maybe only defining an empty object literal for the js scaffolder would be enough? what do you think of that plus a link to https://github.com/form8ion/awesome#languages?

File: README.md [hunk 1]
```diff
@@ -66,14 +66,31 @@ a wrapper.
 
 #### Example
 
-```js
+```javascript
```
Comment (travi) [inline README.md]: was the language set by remark to use the expanded name? i dont really care which is used, but was curious about the change

File: .remarkrc.js [hunk 2]
```diff
@@ -6,4 +6,8 @@ exports.settings = {
   incrementListMarker: false
 };
 
-exports.plugins = ['remark-preset-lint-travi', [require('remark-toc'), {tight: true}]];
+exports.plugins = [
+  'remark-preset-lint-travi',
+  [require('remark-toc'), {tight: true}],
```
Comment (travi) [inline .remarkrc.js]: worth mentioning that i hope to define additional presets for these details at some point, but havent taken the time to figure out the right way to capture the details above

**Comment by travi** [comment]
> plus a link to https://github.com/form8ion/awesome#languages?

i'd be open to also linking to popular plugin examples, like https://github.com/travi/javascript-scaffolder#example, but want to acknowledge that we probably wont think to come and add all over time. that's why i highlight the awesome list first

File: README.md (hunk 1 above)
Comment (trevtrich) [reply README.md]: yup this is what remark printed out

**Comment by trevtrich** [comment]
definitely agree with the concern of adding detailed configs for things like a specific language. i've updated accordingly. still using a real library but i think there's value in that to give a user something they can copy and build themselves if they just want to get started and see something on the cli. the "gateway" 😄

**Pull request merged**